services/movie.py

```diff
@@ -0,0 +1,42 @@
+from django.db.models import QuerySet
+
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: int = None,
+        actors_ids: int = None
+) -> QuerySet[Movie]:
+
+    movies = Movie.objects.all()
+    if genres_ids:
+        movies = movies.filter(genres__id__in=genres_ids).distinct()
+    if actors_ids:
+        movies = movies.filter(actors__id__in=actors_ids).distinct()
+
+    return movies
+
+
+def get_movie_by_id(movie_id: int) -> Movie:
+    return Movie.objects.get(id=movie_id)
+
+
+def create_movie(
+        movie_title: str,
+        movie_description: str,
+        genres_ids: int = None,
+        actors_ids: int = None

```

Comment:
The parameters `genres_ids` and `actors_ids` should be lists of integers, not single integers. Update the type hints to `List[int]` and ensure the function handles lists correctly.